packages/itwinui-react/src/core/Table/Table.test.tsx

```diff
@@ -1661,39 +1661,6 @@ it('should not trigger sorting when filter is clicked', async () => {
   expect(onSort).not.toHaveBeenCalled();
 });
 
-it('should render filter dropdown in the correct document', async () => {
```

Comment:
removed after fighting with it for a week.
<img width="459" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/48f0f8ee-a335-4c3a-8b4e-f480458dbc39">

this test is mocking everything (including me) so not super useful